.github/workflows/black.yml

```diff
@@ -9,8 +8,12 @@ jobs:
     runs-on: ubuntu-20.04
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-      - uses: psf/black@stable
-        with:
-          black_args: ". --check"
\ No newline at end of file
+      - name: Temporarily disabled
```

Comment:
Because this repo will be quite heavily changed by Black, and I wanted to defer that step to a separate PR to avoid overcomplicating this one. Should add a ref to the issue for this.

Reply:
Better yet, I have temporarily disabled the workflow through GitHub. :man_facepalming: